src/main/java/seedu/clirental/CliRental.java

```diff
@@ -3,15 +3,16 @@
 import exceptions.CarException;
 import exceptions.CliRentalException;
 import exceptions.CustomerException;
+import parser.HelpParser;
 import parser.Parser;
 
 public class CliRental {
+    public static final String line = "____________________________________________________________";
```

Comment:
Perhaps this line could be removed? We already have a static method in Parser to print a divider line. Maybe you could just use that instead.

Reply:
ok